sympy/ntheory/modular.py

```diff
@@ -253,3 +255,37 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
+    """
+    Return a list of values whose elements give the remainders
+    in the Cartesian product of the supplied remainders when divided
+    by each of the supplied moduli. If the moduli are not co-prime,
+    you will receive a ValueError.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.modular import crt_cartesian
+    >>> from sympy.utilities.iterables import cartes
+    >>> rem = [[3, 5], [3, 7]]
+    >>> crt_cartesian(rem, [7, 11])
+    [3, 73, 47, 40]
+    >>> [(p%7, p%11) for p in _]
+    [(3, 3), (3, 7), (5, 3), (5, 7)]
+    >>> assert _ == list(cartes(*rem))
```

Comment:
I think `assert` shouldn't be used in `Examples`. Instead show them the outputs of the function.

Reply:
Yes, you can delete this line entirely if this does not look good.
The line above already asserts that the result is correct

Reply:
Ok @czgdp1807 @sylee957 will delete that line.